main.go

```diff
@@ -78,7 +83,13 @@ func main() {
 		log.Infof("Autodiscovered Base Role ARN: %s", config.BaseRoleARN)
 	}
 
-	credsGetter := NewSTSCredentialsGetter(awsSess, config.BaseRoleARN)
+	awsConfigs := make([]*aws.Config, 0)
+	if config.AssumeRoleARN != "" {
+		creds := stscreds.NewCredentials(awsSess, config.AssumeRoleARN)
```

Comment:
I was going to propose resolving this "assume role ARN" value against the base role ARN if the former didn't look like a full ARN, but I noticed that _kiam_ doesn't do that either (see [here](https://github.com/uswitch/kiam/blob/adf03bb349373e22e3d894623870369d45fe2538/pkg/server/server.go#L236) and [here](https://github.com/uswitch/kiam/blob/adf03bb349373e22e3d894623870369d45fe2538/pkg/aws/sts/gateway.go#L86)). Then again, that confused me for a long while when I was trying to deploy _kiam_: some ARNs get resolved against the base, others don't.

Reply:
Fixed, now allows specifying just the role name and prefixes it with base role ARN in that case.